src/components/tl/sharp/tl_sharp.c

```diff
@@ -17,6 +17,11 @@ static ucc_config_field_t ucc_tl_sharp_lib_config_table[] = {
     {"", "", NULL, ucc_offsetof(ucc_tl_sharp_lib_config_t, super),
      UCC_CONFIG_TYPE_TABLE(ucc_tl_lib_config_table)},
 
+    {"USE_INTERNAL_OOB", "n",
```

Comment:
At some point we probably want to make it "y" by default, but need to resolve potential deadlock issue (see below)